cha-ching-app/src/App.js

```diff
@@ -1,17 +1,15 @@
-import Success from "./Success";
-import "./error.css";
 import "./App.css";
+import "./error.css";
 import "./Loading.css";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import ElementsProviderCheckout from "./ElementsProviderCheckout";
-import Hero from "./Hero";
 import ScrollableLandingPage from "./ScrollableLandingPage";
+import Success from "./Success";
 
 function App() {
   return (
     <Router>
       <Routes>
-        <Route path="/hero" element={<Hero />} />
         <Route path="/" element={<ScrollableLandingPage />} />
```

Comment:
this components name could even be LandingPage or just Home :) Since we don't have a nonScrollableLandingPage